installer/photonInstaller.py

```diff
@@ -81,7 +81,7 @@ def create_vmdk_and_partition(config, vmdk_path):
 
     return partitions_data, count == 2
 
-def create_rpm_list_to_copy_in_iso(build_install_option):
+def create_rpm_list_to_copy_in_iso(build_install_option, output_data_path):
```

Comment:
Yes. It looks like its passed in as a parameter to JsonWrapper that translates to filename.